src/parser/jobs/ast/modules/EarthlyStar.tsx

```diff
@@ -89,18 +91,18 @@ export default class EarthlyStar extends Module {
 		/*
 			SUGGESTION: Missed uses
 		*/
-		const holdDuration = this._uses === 0 ? this.parser.fightDuration : this._totalHeld
-		const _usesMissed = Math.floor(holdDuration / (ACTIONS.EARTHLY_STAR.cooldown * 1000))
-		if (_usesMissed > 1 || this._uses === 0) {
+		const holdDuration = this.uses === 0 ? this.parser.fightDuration : this.totalHeld
+		const usesMissed = Math.floor(holdDuration / (this.data.actions.EARTHLY_STAR.cooldown * 1000))
+		if (usesMissed > 1) {
```

Comment:
This should prob be `>= 1` or `> 0` since you have a severity of 1, but it's a tiered suggestion so you can just remove the conditional and it'll only show if they had 1 or more, as your minimum severity is for 1.